Original patch from: gentoo/src/patchsets/glibc/2.9/0051_all_glibc-2.9-add_prio-macros.patch

-= BEGIN original header =-
http://sourceware.org/ml/libc-alpha/2009-01/msg00003.html

From a1737958e074e8f990760acc8abeb0c23715a859 Mon Sep 17 00:00:00 2001
From: Roland McGrath <roland@gnu.org>
Date: Sat, 3 Jan 2009 21:04:01 +0000
Subject: [PATCH] 2009-01-03  Samuel Thibault  <samuel.thibault@ens-lyon.org>

	* bits/resource.h (PRIO_PROCESS, PRIO_PGRP, PRIO_USER): Add macros.
-= END original header =-

diff -durN glibc-2_9.orig/bits/resource.h glibc-2_9/bits/resource.h
--- glibc-2_9.orig/bits/resource.h	2006-03-05 23:48:10.000000000 +0100
+++ glibc-2_9/bits/resource.h	2009-02-02 22:00:38.000000000 +0100
@@ -185,6 +185,9 @@
 enum __priority_which
   {
     PRIO_PROCESS = 0,           /* WHO is a process ID.  */
+#define PRIO_PROCESS PRIO_PROCESS
     PRIO_PGRP = 1,              /* WHO is a process group ID.  */
+#define PRIO_PGRP PRIO_PGRP
     PRIO_USER = 2               /* WHO is a user ID.  */
+#define PRIO_USER PRIO_USER
   };
